build/scripts/update-from.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# update FROM lines in Dockerfiles to the latest base images
+
+# get script
+if [[ ! -f /tmp/updateBaseImages.sh ]]; then 
+    pushd /tmp >/dev/null || exit
+    curl -sSLO https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/master/product/updateBaseImages.sh
+    chmod +x updateBaseImages.sh
+    popd >/dev/null || exit
+fi
+
+# this will attempt to push the change directly, and if it fails, will then 
+# try to generate a PR using hub if installed, or simply create a PR branch in origin
+now="$(date +%s)"
+for d in $(find $(pwd)/dockerfiles -name "*ockerfile" | grep ubi); do 
+    if [[ $(grep access.redhat.com/containers $d) ]]; then 
```

Comment:
IMHO there is something wrong to check redhat repo in scripts

Reply:
no, this is because the logic for staying current requires a comment like this:

```
# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
FROM registry.access.redhat.com/ubi8-minimal:8.2-267 as runtime
```

We could extend this so it supports other registries but then we also need to make sure the skopeo inspection works for those repos...